docs/source/trend_lines.rst

```diff
@@ -0,0 +1,128 @@
+   The mesh is used to identify the initial edges connecting the vertices.
+
+    .. figure:: ./images/trend_lines/trend_step1.png
+            :align: center
+            :width: 300
+
+2. Connections get filtered out based on:
+    - Connection too long, defined by the ``Maximum distance`` parameter
+    - Segments connecting vertices on the same part, as defined by the ``Parts`` data.
+    - [Optional] Segments are filtered based on their deviation from a reference ``Azimuth`` and ``Tolerance`` parameters.
+
+    .. figure:: ./images/trend_lines/trend_step2.png
+            :align: center
+            :width: 300
+
+   The remaining edges move forward to the detection step.
+
+3. The remaining edges are visited exhaustively to find the neighbouring connection that minimizes the tortuosity (:math:`\delta`) of the path.
+
+    .. figure:: ./images/trend_lines/trend_step3.png
+        :align: center
+        :width: 300
+
+        This process is repeated recursively until there is no valid neighbouring edge to form a connection with. The metric used to evaluate the tortuosity is expressed as follows:
+
+    .. math::
+
+        \delta = \theta^{(1-\alpha)} * L
+
+    where the angle :math:`\theta` is the change in direction between the incoming and outgoing segments, :math:`L` is the length of the outgoing segment, and :math:`\alpha` is the `Damping factor <filters>`_.
+    The path is updated with the connection with the lowest :math:`\delta` value.  Note that for
+    :math:`\alpha=1`, only the length of the outgoing segments are considered. This generally increases the number of connections on a trend line, but the path may be more erratic.
+
+
+Example
+-------
+
+This section demonstrates the effect of various detection parameters on a training dataset provided `here <https://github.com/MiraGeoscience/curve-apps/tree/main/edge_detection-assets>`__.
+The goal is to extract trend lines from anomaly picks assigned to a ``Curve`` object. We begin with default parameters and then explore the effect of changing each parameter independently.
+
+.. list-table::
+   :widths: 25 25 25
+   :header-rows: 1
+
+   * - Input data
+     - Default parameters
+     - Result
+
+   * - .. figure:: ./images/trend_lines/example_data.png
+            :align: center
+            :width: 300
+     - .. figure:: ./images/trend_lines/trend_lines_ui.png
+            :align: center
+            :width: 300
+     - .. figure:: ./images/trend_lines/example_no_filter.png
+            :align: center
+            :width: 300
+
+Note the we have recovered line paths that connect our input data, but many of those segments are not desirable. They are either
+connecting points that are too far apart or allowed to change direction rapidly. We can improve the results by adjusting the parameters.
+
+.. list-table::
+   :widths: 25 25
+   :header-rows: 1
+
+   * - Detection parameters
+     - Result
+
+   * - .. figure:: ./images/trend_lines/example_filtered_ui.png
```

Comment:
I would align these vertically too, with last sentence in between the two images.